style.css

```diff
@@ -1 +1,56 @@
 /*  Add your styling here */
+body  {
+    display: grid;
+    grid-template-areas: 1fr 2fr 4fr 3r;
+    background-color: #ffe6ff;
+}
+
+.nav-bar > ul {
+   display: inline;
+   padding: 100px 6em 6em 7em;
```

Comment:
I suggest using rems instead of ems in general. 

Ems are a relational scaling element that can create huge problems when users scale up their text. Lots of users do this.